plugins/modules/lightsail.py

```diff
@@ -10,7 +10,7 @@
 DOCUMENTATION = '''
 ---
 module: lightsail
-version_added: 1.0.0
+version_added: 1.1.0
```

Comment:
```suggestion
version_added: 1.0.0
```

This is the version the module was added to the collection, and doesn't need updating